ext/js/data/anki-note-builder.js

```diff
@@ -54,7 +54,6 @@ export class AnkiNoteBuilder {
         fields,
         tags = [],
         requirements = [],
-        checkForDuplicates = true,
```

Comment:
Since this is removed, can this be removed from the TS definition from the type?

Reply:
done